src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,53 @@
+package core.basesyntax.model;
+
+public class FruitTransaction {
+    private Operation operation;
+    private Fruit fruit;
+    private int quantity;
+
+    public FruitTransaction(String operation, String fruit, String quantity) {
```

Comment:
```suggestion
    public FruitTransaction(Operation operation, String fruit, int quantity) {
```
Из конструктора нужно убрать всю логику